### ISSUE TITLE
Refine menu/burger focus state

### ISSUE THREAD

**Issue by youriwims** [mozilla/donate-wagtail#340]
### Issue: Focus state not clear

**Element path:** `.burger`

**Snippet:** `<button class="burger" data-menu-toggle="" aria-haspopup="true" aria-label="Menu toggle"></button>`
![Screen Shot 2019-10-01 at 11 07 58 PM](https://user-images.githubusercontent.com/36484608/66015384-5a284680-e4a0-11e9-8722-672a2a5787ae.png)

**How to fix:**
Fix any of the following:
- Remove `outline: 0` from `.burger`

====

More Info:

Comments: The menu appears low-opacity when it's focused for some reason—this issue is primarily in Google Chrome. In Firefox, there a dotted line present around the menu icon.
Resources: n/a

**Comment by youriwims** [comment]
Also, pretty sure the burger bars should be `#000000` at all times. @beccaklam can you confirm pls?

**Comment by nicklee** [comment]
We'd previous removed outline and added an opacity hover/active/focus state as the outline wasn't a desired look.

Happy to update, but guess we just need some guidance on what the new hover/active/focus state should be @beccaklam? I guess just change it from black to another colour could work? The current site's hamburger doesn't have anything.

**Comment by beccaklam** [comment]
I think as it currently stands we don't have a hover state for the hamburger menu. The only feedback we have is that a user's cursor turns into a hand pointer indicating that the icon is clickable. And yes the icon should be #000000 at all times. Is that enough definition @nicklee ? Perhaps in the future we will had more feedback but let's keep it consistent with what we have on our other web properties for now.

**Comment by nicklee** [comment]
Hi @beccaklam, I believe @youriwims point is that users using a keyboard to navigate currently get no feedback when they focus the burger, so this ideally needs to change. Feels like something we should fix as it's the main navigation and first item users will tab on to (try hitting tab, it feels like nothing has happened on the live site, on the new staging site we've reduced the opacity, but it's very subtle).

**Comment by beccaklam** [comment]
Hi @nicklee I'll have to talk to the rest of the design team about it since it's changing our style guide. I'm thinking perhaps we can just lighten the opacity on hover.

**Comment by beccaklam** [comment]
@nicklee After talking to the rest of the design team we've decided to change the colour of the hamburger nav to #666666 on hover. Let me know if you have any questions!

**Comment by nicklee** [comment]
Focus and hover set to #666666, hamburger made solid black:

https://github.com/mozilla/donate-wagtail/pull/383

**Comment by youriwims** [comment]
In order to close this ticket, we need to remove [this line of code](https://github.com/mozilla/donate-wagtail/pull/383/files#diff-d94be0022de14e33c2f931d322b43222R11).

**Comment by mmmavis** [comment]
Seems like this has been done already. Closing.
